This causes the zlib decls to be stdcall, but it is not built that way

--- origsrc/openexr-1.7.1/IlmImf/ImfPxr24Compressor.cpp	2012-07-17 18:27:27.000000000 -0500
+++ src/openexr-1.7.1/IlmImf/ImfPxr24Compressor.cpp	2013-11-26 20:46:31.018331700 -0600
@@ -62,7 +62,6 @@
 //	string of bytes is compressed with zlib.
 //
 //-----------------------------------------------------------------------------
-#define ZLIB_WINAPI 
 
 #include <ImfPxr24Compressor.h>
 #include <ImfHeader.h>
